indexer.go

```diff
@@ -1,31 +1,71 @@
 package indexer
 
+import (
+	"github.com/pokt-foundation/pocket-indexer/types"
+)
+
 // Provider interface of needed provider functions
 type Provider interface {
 	GetBlockHeight() (int, error)
-	// TODO: add missing functions
+	GetBlock(blockHeight int) (*types.Block, error)
```

Comment:
These should return the values they return from pocket go, that's what the provider will return